downloads/swan-lake-release-notes/swan-lake-preview2/RELEASE_NOTE.md

```diff
@@ -226,30 +226,30 @@ public function main() {
 
 ### HTTP
 
-The redirect client has been improved to support temporary and permanent redirects, with the original request’s HTTP method. A new `allowAuthHeaders` field was introduced in the redirect configuration to get user consent before repeating sensitive data on subsequent requests.
+The redirect client has been improved to support temporary and permanent redirects with the original request’s HTTP method. A new `allowAuthHeaders` field was introduced in the redirect configuration to get user consent before repeating sensitive data on subsequent requests.
 
 ### Log
 
-A function  log:setModuleLogLevel has been introduced  to allow users to set the module log level through the Log API.
+A `log:setModuleLogLevel` function has been introduced to allow users to set the module log level through the Log API.
```

Comment:
```suggestion
The new `log:setModuleLogLevel` function allows users to set the module log level through the Log API.
```

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/1072.